agv/docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ros:melodic-ros-base
```

Comment:
I think this image is based on Ubuntu:bionic, so it is better to replace it with raspbian:buster. How about you?

Reply:
IMO, the difference of linux distribution is not a problem because all we want is for the binary to work well. And it's  more convenient to use official image because we don't build and maintain the ros packages.